### ISSUE TITLE
Map download speed fluctuations

### ISSUE THREAD

**Issue by sonora** [osmandapp/OsmAnd#15272]
I sporadically observe slow download speeds when updating maps. In order to debug this, I have just tried an "Update all" in OsmAnd and observe that the download speed for each map file map file seems to remain roughly constant, but between map files varies by as much as an order of magnitude: sometimes 30Mbps, but for other files dropping to as low as 3Mbps. Do we maybe have an issue with a server or a load balancer?

**Comment by vshcherb** [comment]
Yes, we now redirecting too many german users to 1 server, cause other servers aren't always supporting german IPs (there is a dicussion /issue here). 

Usually on 8-9 day of the month we are under heavy load, now all servers are free. We did couple of optimization to speed up maps spreading and avoid double map downloads. Next month should be better but ultimate solution is to have Cloud 1-2 day servers that we can redistribute monthly maps and kill unnecessary servers.
